seed/views/geocode.py

```diff
@@ -82,6 +82,10 @@ def confidence_summary(self, request):
                     id__in=tax_lot_ids,
                     geocoding_confidence__startswith='Low'
                 )),
+                'manual': len(TaxLotState.objects.filter(
```

Comment:
Part of the fix for manually geocoded taxlots not being noted when "Geocoding Selected" from the inventory list page.